source/geometries/Next100EnergyPlane.cc

```diff
@@ -105,20 +104,8 @@ namespace nexus {
     GeneratePositions();
 
     /// Copper Plate ///
-    G4Tubs* copper_plate_origin_solid =  new G4Tubs("COPPER_PLATE_ORIGIN", 0., copper_plate_diam_/2.,
-                                                    copper_plate_thickn_/2., 0., twopi);
-
-    /// Hole for gas flow ///
-    G4double gas_hole_diam   = 12.  * mm;
```

Comment:
Removing this hole changes the volume. Have you assessed the impact on the background simulation? Also, there may be gammas penetrating through the hole that now would be blocked. I'm note sure we should remove it.